frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/mirroring-card/mirroring-card.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+
+import { ocsResource } from '../../../../resources';
+import '../../common/storage-efficiency/storage-efficiency-card.scss';
+
+export const MirroringItemBody: React.FC<MirroringItemBodyProps> = React.memo(
+  ({ title, isLoading, error, children }) => {
+    const { t } = useTranslation();
+
+    let status: React.ReactElement;
+
+    if (isLoading) {
+      status = <div className="skeleton-text ceph-storage-efficiency-card__item-body--loading" />;
+    } else if (error) {
+      status = (
+        <span className="co-dashboard-text--small text-muted">
+          {t('ceph-storage-plugin~Not available')}
+        </span>
+      );
+    } else {
+      status = <span className="ceph-storage-efficiency-card__item-text">{children}</span>;
+    }
+    return (
+      <div className="co-inventory-card__item">
+        <div className="ceph-storage-efficiency-card__item-title">{title}</div>
+        <div
+          className="ceph-storage-efficiency-card__item-status"
+          data-test={`${title}-mirroring-card-status`}
+        >
+          {status}
+        </div>
+      </div>
+    );
+  },
+);
+
+const MirroringCard: React.FC<DashboardItemProps> = ({
+  watchK8sResource,
+  stopWatchK8sResource,
+  resources,
+}) => {
+  const { t } = useTranslation();
+
+  React.useEffect(() => {
+    watchK8sResource(ocsResource);
+    return () => {
+      stopWatchK8sResource(ocsResource);
+    };
+  }, [watchK8sResource, stopWatchK8sResource]);
+
+  const ocs = resources?.ocs;
+  const mirroringStatus: boolean = ocs?.data?.[0]?.spec.hasOwnProperty('mirroring');
+  const mirroringStatusProps = {
+    title: t('ceph-storage-plugin~Mirroring Status'),
+    isLoading: !ocs?.loaded,
+    error: !!ocs?.loadError,
+  };
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>{t('ceph-storage-plugin~Mirroring')}</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody className="co-dashboard-card__body--no-padding">
+        <MirroringItemBody {...mirroringStatusProps}>
+          <div className="co-dashboard-text--small text-muted">
+            {mirroringStatus ? t('ceph-storage-plugin~Enabled') : t('ceph-storage-plugin~Disabled')}
+          </div>
```

Comment:
```suggestion
        {mirroringStatus ? t('ceph-storage-plugin~Enabled') : t('ceph-storage-plugin~Disabled')}
```


Better move this div and class part inside MirroringItemBody

Reply:
I find this way more generic...
If some other Item wants to display some value other than Enable/Disable they cab be passed as a children to the component.
Tried to do similar to what we are doing in (https://github.com/openshift/console/pull/9423)
^^ https://github.com/openshift/console/pull/9587#discussion_r674857220.

Reply:
Should I still make the change ?

Reply:
I got your point, one better way is just like <DetailItem> you can pass the valueClass name as on property and you can use it inside MirroringItemBody. So can avoid <div className="co-dashboard-text--small text-muted"> here.

Reply:
you can apply for child class name by passing inside MirroringItemBody

Reply:
ack... thanks for the clarification... will update this...